### ISSUE TITLE
Training loss does not go down while using TGCN

### ISSUE THREAD

**Issue by Vidyaranya** [dxli94/WLASL#23]
Congrats on the excellent repo. I was training using the TGCN module. I had placed all the pose keypoints and correctly changed the config parameters in train_tgcn.py. However, I see that the training loss is not at all decreasing. May I know why might that be happening?

**Comment by dxli94** [comment]
Have you figured out the problem? Did you manage to solve it?

**Comment by Vidyaranya** [comment]
It seems to be coming from the batch size. Using a high batch size like 1024 even with a higher learning rate seems to not give the accuracy. However, using the batch size exactly as the one provided in the config files works fine. Happy to close the issue and thanks for your reply.
